llmfoundry/utils/data_prep_utils.py

```diff
@@ -131,7 +132,8 @@ def __iter__(self):
                 output_filename = os.path.join(
                     self.output_folder,
                     object_name.strip('/'),
-                )
```

Comment:
just for my understanding, why don't we need to rename these files from the temp dir? I guess if someone hasn't specified the output_folder then it doesn't matter

Reply:
ah like rename them with object_name? That was causing the name too long issue.